.docs/content/guide/3.end2end.md

```diff
@@ -0,0 +1,72 @@
+<!-- @case-police-ignore Api -->
+<!-- @case-police-ignore Sdk -->
+
+# Build and configure your own client/worker test
+
+You can also develop your own client to submit jobs to ArmoniK and invoke desired services  which are embedded in the worker-shared library.
+
+## Client test
+
+The SessionService class in ```ArmoniK::Sdk::Client``` namespace is used to create a client. This class essentially contains methods for sending tasks to ArmoniK and retrieving the results. These methods are ```Submit``` and ```WaitResults``` respectively.
+
+### Configure client
+
+The constructor of the SessionService class takes as arguments a properties and a logger objects. The properties object as defined in ```ArmoniK::Sdk::Common``` namespace contain the configuration and a session task options objects.
```

Comment:
a property or properties?

Reply:
I wanted to keep the name of the variable in the code, so its properties. But I will highlight it to avoid confusion.